server/testdb/seed.go

```diff
@@ -36,7 +36,7 @@ func SeedApplication(db datastore.DatabaseClient, g *datastore.Group, uid, title
 
 	// Seed Data.
 	appRepo := db.AppRepo()
-	err := appRepo.CreateApplication(context.TODO(), app)
+	err := appRepo.CreateApplication(context.TODO(), app, app.GroupID)
```

Comment:
@danvixent This defeats the purpose. Let the seed functions receive a groupID and use the default group ID to ensure things work well.